tests/integration/CapabilityTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Automattic\LegacyRedirector\Tests;
+
+use Automattic\LegacyRedirector\Capability;
+
+/**
+ * CapabilityTest class
+ */
+class CapabilityTest extends TestCase {
+
+	protected static $initialized = false;
+
+	public function setUp() {
+		if ( ! self::$initialized ) {
+			$capability = new Capability();
+			$capability->register();
+
+			self::$initialized = true;
+		}
+	}
+
+	/**
+	 * Test administrator capabilities for Capability::MANAGE_REDIRECTS_CAPABILITY
+	 *
+	 * @return void
+	 */
+	public function test_admin_capability() {
+
+		$user_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		$user    = wp_set_current_user( $user_id );
```

Comment:
Instead of testing a _new_ administrator, can you also check to see if an existing administrator account has got the cap? (Perhaps another reason to move the `Capability->register()` to these test methods so you can do a before and after check.

Reply:
I modified the tests accordingly, added also an unregister method with the relevant test in the Capability section.